translation/sr/brankokovacevic/sutta/mn/mn29_translation-sr-brankokovacevic.json

```diff
@@ -0,0 +1,115 @@
+{
+  "mn29:0.1": "Srednja Zbirka 29",
+  "mn29:0.2": "Dugi govor o poređenju sa središtem stabla",
+  "mn29:1.1": "Ovako sam čuo.",
+  "mn29:1.2": "Jednom je Blaženi boravio kraj Rāđagahe, na Lešinarevoj hridi, nedugo pošto je Devadatta otišao.",
+  "mn29:1.3": "Tu se Blaženi, u vezi sa Devadattom, ovako obrati monasima:",
+  "mn29:2.1": "„Monasi, recimo da neki čovek iz poverenja napusti domaćinski život i ode u beskućnike, razmišljajući:",
+  "mn29:2.2": "'Žrtva sam rođenja, starosti i smrti, žalosti, naricanja, bola, tuge i očaja; žrtva sam patnje, plen patnje.",
+  "mn29:2.3": "Sigurno da je moguće stići na kraj čitavog tog okeana patnje.'",
+  "mn29:2.4": "I kad je otišao u beskućnike, stiče dobitak, poštovanje i dobar glas.",
+  "mn29:2.5": "Zadovoljan je tim dobitkom, poštovanjem i dobrim glasom, njegova namera biva ostvarena.",
+  "mn29:2.6": "Na osnovu toga sam sebe hvali, a druge kritikuje ovako:",
+  "mn29:2.7": "'Stekao sam dobitak, poštovanje i dobar glas, ali ovi drugi monasi su nepoznati, niko za njih i ne zna.'",
+  "mn29:2.8": "Tako postaje opijen dobitkom, poštovanjem i dobrim glasom, te je sve više nepažljiv, osvaja ga nepažljivost. Pošto je nepažljiv, živi u patnji.",
+  "mn29:2.9": "Zamislite čoveka kojem treba dobra drvena građa, traga za dobrom drvenom građom i tragajući za zdravim drvetom, naiđe na veliko drvo snažnog debla. Zanemarivši srž stabla, njegov meki deo, unutrašnju koru i spoljašnju koru, naseče on grane i lišće, te ih ponese misleći da je to srž stabla.",
+  "mn29:2.10": "Onda čovek dobroga vida, kad ga sretne, kaže:",
+  "mn29:2.11": "'Ovaj dobri čovek nije znao šta je srž stabla, njegov meki deo, unutrašnja kora, spoljašnja kora, niti grane i lišće.",
+  "mn29:2.12": "Pošto mu je bila potrebna dobra drvena građa, tragajući za dobrom drvenom građom, naišao je na veliko drvo snažnog debla. Zanemarivši srž stabla, njegov meki deo, unutrašnju koru i spoljašnju koru, nasekao je grane i lišće te ih poneo misleći da je to srž stabla.",
+  "mn29:2.13": "Šta god da je ovaj dobri čovek mislio da napravi od dobre drvene građe, ovo mu neće poslužiti za to.'",
+  "mn29:2.14": "Isto tako, monasi, kad čovek iz poverenja napusti domaćinski život...",
```

Comment:
@aleksabisercic please update all occurences of dots to Unicode ellipsis
https://github.com/suttacentral/bilara-data/wiki/SuttaCentral-translation-style-guide#little-details